core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2341,6 +2365,15 @@ public String toString() {
         "alluxio.security.group.mapping.class";
     public static final String SECURITY_LOGIN_USERNAME = "alluxio.security.login.username";
 
+    //
+    // JVM related properties
+    //
+    public static final String JVM_MONITOR_WARN_THRESHOLD_MS = "jvm.monitor.warn.threshold.ms";
+    public static final String JVM_MONITOR_INFO_THRESHOLD_MS = "jvm.monitor.info.threshold.ms";
+    public static final String JVM_MONITOR_SLEEP_INTERVAL_MS = "jvm.monitor.sleep.interval.ms";
+    public static final String JVM_MONITOR_MASTER_ENABLE = "jvm.monitor.master.enable";
+    public static final String JVM_MONITOR_WORKER_ENABLE = "jvm.monitor.worker.enable";
```

Comment:
Are these standard options? If not, could you change them to Alluxio style? ie. `alluxio.master.jvm.monitor.enabled`?